includes/class-wcal-admin-notice.php

```diff
@@ -77,4 +77,32 @@ public static function wcal_show_db_update_notice() {
 			}
 		}
 	}
+	/**
+	 * Enabling this setting will allow users registered on the website to access it via reminder email links without needing to login
+	 *
+	 * @since 5.16.0
+	 *
+	 * @hook admin_notices
+	 */
+	public static function wcal_show_auto_loggin_notice() {
+
+		if ( isset( $_GET['page'] ) && 'woocommerce_ac_page' === $_GET['page'] && isset( $_GET['action'] ) && 'emailsettings' === $_GET['action'] ) { // phpcs:ignore WordPress.Security.NonceVerification
+
+			if ( 'on' === get_option( 'wcal_auto_login_users', 'on' ) ) {
+				?>
+					<div id='wcal_cron_notice' class='is-dismissible notice notice-info wcal-cron-notice'>
```

Comment:
@nidhitatosaniya - can u pls add code to make the notice dismissible? i.e. if the user dismisses the notice we should save that and ensure the notice is not displayed unless the field value is changed again.